iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractLoadingLimits.java

```diff
@@ -95,4 +103,35 @@ public double getTemporaryLimitValue(int acceptableDuration) {
         TemporaryLimit tl = getTemporaryLimit(acceptableDuration);
         return tl != null ? tl.getValue() : Double.NaN;
     }
+
+    private void checkTemporaryLimits() {
+        // check temporary limits are consistent with permanent
+        if (LOGGER.isDebugEnabled()) {
+            double previousLimit = Double.NaN;
+            boolean wrongOrderMessageAlreadyLogged = false;
+            for (LoadingLimits.TemporaryLimit tl : temporaryLimits.values()) { // iterate in ascending order
```

Comment:
Where do we have the rule (or the check) that values must be ordered? Because it was a question I had, without any clear answer that we have a check on that.

Reply:
Temporary limits are automatically ordered by descending order of their acceptable durations (it's the `temporaryLimit` variable of `AbstractLoadingLimitsAdder`, a `TreeMap` initialized with the `AbstractLoadingLimitsAdder.ACCEPTABLE_DURATION_COMPARATOR` comparator, which is responsible for that).
Therefore, acceptable durations are ordered by descending order, but nothing ensures that values are ordered by ascending order.